ui/app/adapters/config-ui/message.js

```diff
@@ -4,26 +4,14 @@
  */
 
 import ApplicationAdapter from '../application';
-import { encodePath } from 'vault/utils/path-encoding-helpers';
 
 export default class MessageAdapter extends ApplicationAdapter {
-  getCustomMessagesUrl(id) {
-    let url = `${this.buildURL()}/config/ui/custom-messages`;
-
-    if (id) {
-      url = url + '/' + encodePath(id);
-    }
-    return url;
+  pathForType() {
+    return 'config/ui/custom-messages';
   }
 
-  query(store, type, query) {
+  query(store, type, query, recordArray, adapterOptions) {
```

Comment:
I don't think you need to pass in these two additional args since all you're using here is `query`?

[docs](https://api.emberjs.com/ember-data/5.3/classes/Adapter/methods/query?anchor=query)